framework/ui/iconcodes.h

```diff
@@ -58,17 +55,17 @@ class IconCode
         LOOP = 0xEF1F,
         METRONOME = 0xEF20,
         TUNING_FORK = 0xEF21,
-        FILE_NEW = 0xEF22,
-        FILE_OPEN = 0xEF23,
-        FILE_SHARE = 0xEF24,
-        FILE_CLOUD = 0xEF25,
+        NEW_FILE = 0xEF22,
+        OPEN_FILE = 0xEF23,
+        SHARE_FILE = 0xEF24,
+        CLOUD_FILE = 0xEF25,
```

Comment:
Where do the names come from?

Reply:
They came from the Martin's icon font

Reply:
I think you need to add comments, what kind of codes are they, where they come from, how to add another one, and so on.